Makefile

```diff
@@ -596,6 +598,10 @@ cinder_deploy: input cinder_deploy_prep ## installs the service instance using k
 	$(eval $(call vars,$@,cinder))
 	oc kustomize ${DEPLOY_DIR} | oc apply -f -
 
+.PHONY: cinder_deploy_validate
```

Comment:
please see jgilaber review @ https://github.com/openstack-k8s-operators/install_yamls/pull/59/files 

update the names 

2 targets one for all the things with kuttl

another with just kuttl

Reply:
ack, added the 2 targets in https://github.com/openstack-k8s-operators/install_yamls/pull/69/commits/c3883daf0069fce8d105fa26638e3151dee7b3a2